docs/statistics/hypothesis/Ttest.md

```diff
@@ -1,3 +1,724 @@
+    \]
+
+### Applying the Test
+
+1. **Set the Hypotheses**:
+    - **Null hypothesis (H~0~)**: The means of the two groups are equal (\(\mu_1 = \mu_2\)).
+    - **Alternative hypothesis (H~1~)**: The means of the two groups are not equal (\(\mu_1 \ne \mu_2\)).
+2. **Collect Data**: Obtain independent random samples from both groups.
+3. **Calculate the Sample Means (\(\bar{x}_1\) and \(\bar{x}_2\))**.
+4. **Compute the Sample Variances (\(s_1^2\) and \(s_2^2\))**.
+5. **Calculate the t-value** using the formula above.
+6. **Determine Degrees of Freedom** using the Welch-Satterthwaite equation.
+7. **Find the p-value**: Use the t-distribution table or statistical software.
+8. **Make a Decision**: If the p-value is less than your significance level (e.g., 0.05), reject the null hypothesis.
+
+
+
+???+ example "Example: Comparing Teaching Methods"
+
+    A researcher wants to determine if two different teaching methods lead to different student performance levels. They randomly assign students to two groups: one uses **Method A**, and the other uses **Method B**. After a semester, both groups take the same standardized test.
+
+    **Assumptions**:
+
+    - Significance level \( \alpha = 0.05 \)
+    - Two-tailed test: Testing for any difference in means
+
+    ---
+
+    - **Set the hypotheses**:
+        - **Null hypothesis (H~0~)**: \(\mu_1 = \mu_2\) (no difference in mean scores)
+        - **Alternative hypothesis (H~1~)**: \(\mu_1 \ne \mu_2\) (difference in mean scores)
+
+    ---
+
+    - **Collect sample data**:
+
+        ```py
+        # Generate Data
+        import numpy as np
+
+        np.random.seed(42)  # for reproducibility
+
+        # Group sizes
+        n1 = 30  # Method A
+        n2 = 35  # Method B
+
+        # Simulate test scores
+        mean1, std1 = 75, 10  # Method A
+        mean2, std2 = 80, 12  # Method B
+
+        scores1 = np.random.normal(mean1, std1, n1)
+        scores2 = np.random.normal(mean2, std2, n2)
+        ```
+
+    ---
+
+    ??? example "Manual Calculation"
+
+        - **Calculate sample means and variances**:
+
+            ```py
+            mean1 = np.mean(scores1)
+            mean2 = np.mean(scores2)
+            var1 = np.var(scores1, ddof=1)
+            var2 = np.var(scores2, ddof=1)
+
+            print(f"Mean of Method A: {mean1:.2f}")
+            print(f"Mean of Method B: {mean2:.2f}")
+            print(f"Variance of Method A: {var1:.2f}")
+            print(f"Variance of Method B: {var2:.2f}")
+            ```
+
+            ```title=">>> Output"
+            Mean of Method A: 73.12
+            Mean of Method B: 78.01
+            Variance of Method A: 81.00
+            Variance of Method B: 119.45
+            ```
+
+        - **Calculate the t-value**:
+
+            \[
+            t = \frac{\bar{x}_1 - \bar{x}_2}{\sqrt{\frac{s_1^2}{n_1} + \frac{s_2^2}{n_2}}} = \frac{74.86 - 80.99}{\sqrt{\frac{75.26}{30} + \frac{143.03}{35}}} \approx -2.09
```

Comment:
I think some values need to be replaced to align with the calculation in the code snippet above? 😄

```suggestion
            t = \frac{\bar{x}_1 - \bar{x}_2}{\sqrt{\frac{s_1^2}{n_1} + \frac{s_2^2}{n_2}}} = \frac{73.12- 78.01}{\sqrt{\frac{81}{30} + \frac{119.45}{35}}} \approx -1.98
```